Create 2025-06-19-increased-blob-size-limits-in-Workers-Analytics.mdx (#23129)
cloudflare/cloudflare-docs: src/content/changelog/workers/2025-06-20-increased-blob-size-limits-in-Workers-Analytics.mdx

---
title: Increased blob size limits in Workers Analytics Engine
description: We’ve increased the total allowed size of blob fields per request from 5 KB to 16 KB in the Workers Analytics Engine.
products:
  - workers
date: 2025-06-20T11:29:00Z
---
import { TypeScriptExample } from "~/components";

> We’ve increased the total allowed size of [`blob`](/analytics/analytics-engine/get-started/#2-write-data-points-from-your-worker) fields per request from **5 KB to 16 KB** in the Workers Analytics Engine.

Reviewer: ```suggestion
We’ve increased the total allowed size of [`blob`](/analytics/analytics-engine/get-started/#2-write-data-points-from-your-worker) fields on data points written to [Workers Analytics Engine](/analytics/analytics-engine/) from **5 KB to 16 KB**.
```

question - is the limit actually _per request_ or per _datapoint_?

When I read the docs it's ambiguous to me

https://developers.cloudflare.com/analytics/analytics-engine/limits/

Because these could refer to the incoming request to the Worker, or refer to `writeDataPoint` call as a "request"

When I saw this I kept assuming it was per datapoint, since that seemed logical and then paused.

Reviewer: If it's actually per _request_ that's much harder to reason about when writing code around this

whatever we dig up should probably clarify in the existing docs

Reviewer: If it's per datapoint then the line above needs to be updated
